### ISSUE TITLE
Work with usertoken on Python 3.7?

### ISSUE THREAD

**Issue by 323Works** [sjmh/pyqb#13]

**Comment by mirandamo-lqd** [comment]
wondering as well

**Comment by mirandamo-lqd** [comment]
following up

**Comment by mkarrlp** [comment]
I'm struggling even getting this to connect at all. It always tells me invalid credentials even though they are correct. If I try to use a user_token it just says my ticket is invalid.. It should be skipping the ticket and using the user_token, right? I can't even get this to authenticate at all.

**Comment by mirandamo-lqd** [comment]
i was able to get it connect 
```
client = pyqb.Client(url=URL, user_token=TOKEN)
```

**Comment by mkarrlp** [comment]
That is exactly what I am doing. There is no error with that command. It is when I go to do the next command like doquery, that is when the error happens. It tells me "pyqb.ResponseError: Received err #24 : Invalid Application Token", I know the user_token is valid and is assigned correctly to the right application from what I understand. Not sure what else to try, me and another user are running into this same problem.

**Comment by mirandamo-lqd** [comment]
i believe thats your QB's thing, did you add the app you wanted to connect to your usertoken?

**Comment by mkarrlp** [comment]
Yes my User Token is setup correctly for the app I am trying to use.

To me the error appears to state it is attempting to use an Application Token rather than my supplied User Token(but the error message may be generic). I have even tried to use an Application Token in its place(based on the error) but that does nothing.

I'm going to keep trying to figure this out. Thanks for the help.

**Comment by mirandamo-lqd** [comment]
![image](https://user-images.githubusercontent.com/56565582/71740638-61098100-2e22-11ea-81ac-d596d50b4f0d.png)

TOKEN = ur usertoken that has connected to the same app
URL = "https://urdomain.quickbase.com"
client = pyqb.Client(url=URL, user_token=TOKEN)

hope this helps

**Comment by mkarrlp** [comment]
I appreciate the help but I have made sure the user token is attached to the right applications. I am not having trouble understanding that part, it is done correctly.

I won't bore you with the details but I think my issue was I was using http instead of https. Such a simple thing made it work now.

Edit: I lied, the doquery still fails for the same reason... But I can at least authenticate now lol. I think switching to using straight Requests is going to be easier than this library but again thanks for the help.

**Comment by mirandamo-lqd** [comment]
responses = client.doquery(qid = 1, database = TABLEID, fmt=True, fields= {3,6,27,28,25,26,116,138,92,59,15,67,83,87,166,209,142,174,99,98,97,96,103,176,263})
